bep032tools/generator/tests/test_BEP032Generator.py

```diff
@@ -1,4 +1,5 @@
 import os
+import pathlib
```

Comment:
You are not using this import in your test. Maybe you forgot this in your cleanup?